src/include/daos_pool.h

```diff
@@ -49,11 +49,12 @@ typedef struct {
 	int			foo;
 } daos_target_perf_t;
 
-enum {
+typedef enum {
 	DAOS_MEDIA_SCM	= 0,
 	DAOS_MEDIA_NVME,
+	DAOS_MEDIA_OPTANE,
```

Comment:
Maybe we can replace the 'DAOS_MEDIA_NVME' to 'DAOS_MEDIA_NVME_PERF' and 'DAOS_MEDIA_NVME_CAPACITY'?